modules/basic-skills/src/backend/slot.ts

```diff
@@ -129,11 +129,20 @@ const createNodes = data => {
           name: 'builtin/setVariable {"type":"temp","name":"alreadyExtracted","value":"true"}'
         }
       ],
-      onReceive: undefined,
+      onReceive: [
+        data.validationAction && {
+          type: sdk.NodeActionType.RunAction,
+          name: `${data.validationAction} {}`
+        }
+      ],
       next: [
         {
-          condition: 'true',
+          condition: ' (temp.valid === undefined || temp.valid == "true")',
```

Comment:
=== ?

Reply:
we want to check that the value is truthy not that it is exactly equal to true. I used the same condition as the one on line 91 to make sure it acts the same way

Reply:
It is considered bad practice in JavaScript to triple equals against the undefined global (there are some weird cases).
Using `typeof x === "undefined"` is better practice.

Reply:
@JustusNBB can you elaborate on that ? I know there's a difference between `null` and `undefined` with the triple equality, but what are those other casees you are talking about? 
`undefined === undefined` = true
`null === undefined` = false
`null == undefined` = true

Personally i'm  okay with both usage, like `val !== undefined && val !== null` or `val != undefined` to handle both